src/Files/allEvents.js

```diff
@@ -1,4 +1,73 @@
-export default [
+import jQuery from 'jquery';
+
+window.$ = window.jQuery = jQuery;
+
+let eventlist = []
+
+// Stuff to hook it up to the Google Cloud Api, API_KEY are unique
+// Generating an API Key: https://developers.google.com/workspace/guides/create-project
+const API_KEY = 'AIzaSyDmgrkOrEu0ZFMxT8ra1H42evtCDoKXhA8';
+
+// This needs to be set to a PUBLIC Google calendar
+// Also this type of access is read-only so you can't push events from the cs club website if you wanted to try that
+// How to get calendar ID: https://yabdab.zendesk.com/hc/en-us/articles/205945926-Find-Google-Calendar-ID
+const CAL_ID = '25h073198b7tpg7qcj6uhu1t8k@group.calendar.google.com';
+
+let months = ["JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC"]
+let newEvents = [];
+
+    jQuery.ajax({
+        url:"https://www.googleapis.com/calendar/v3/calendars/" + CAL_ID + "/events?singleEvents=true&orderBy=startTime&sortOrder=ascending&timeMin=" + (new Date()).toISOString() + "&key=" + API_KEY,
+        type: "GET",
+        dataType: "json",
+        async:false,
+        // async lets the data stay saved after the inital call, otherwise the data will be removed after the success call
+        success: function(data) {
+          console.log(data);
+          eventlist = data.items;
+          console.log(eventlist);
+        }
+          });
+
+
+          for(var i = 0; i < eventlist.length; i++){
+            let hours = parseInt(eventlist[i].start.dateTime.substring(11, 13));
```

Comment:
In JavaScript we usually write the for-loop like this:` eventlist.map((item) => {do whatever you need to do and return a new item in the end}) `What this means is that for each element `item` in` eventlist`, we do something and return a new item. The returned item can be any type. And in the end the function will return an array of all new items. If you don't want to return anything, you can do `eventlist.forEach((item) => {})`. It does the same trick as map but it doesn't require you to return anything (so in this case you can just push the formatted item to `newEvents` instead of returning it). You probably will learn it in COMP215, so just a preview on it. No need to change it for now.